extension/manifest.json

```diff
@@ -27,6 +27,11 @@
      "run_at"     : "document_end",
      "all_frames" : true
    }],
+   "commands": {
+    "current-tab-link": {
+      "description": "current tab in default format"
+    }
+   },
```

Comment:
Results in:
![image](https://cloud.githubusercontent.com/assets/6391742/16213759/2182777a-370f-11e6-860f-36b207d3b252.png)